src/dotnet/commands/dotnet-test/BaseDotnetTestRunner.cs

```diff
@@ -54,10 +53,10 @@ private int DoBuildTestProject(DotnetTestParams dotnetTestParams)
                 strings.Add(dotnetTestParams.Output);
             }
 
-            if (!string.IsNullOrEmpty(dotnetTestParams.Runtime))
+            if (!string.IsNullOrEmpty(projectContext.RuntimeIdentifier))
             {
                 strings.Add("--runtime");
-                strings.Add(dotnetTestParams.Runtime);
+                strings.Add(projectContext.RuntimeIdentifier);
```

Comment:
Isn't this ignoring what the user put for `--runtime` when they called `dotnet test`?  Is that intended?  If a user says `dotnet test --runtime Banana1.0` shouldn't we be building for Banana1.0?